readme.md

```diff
@@ -51,6 +51,17 @@ git clone --recursive https://github.com/Azure/azure-iot-sdk-csharp.git
+|-----------------------------|--------------------|--------------------|--------------------|--------------------|--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| TPM Individual Enrollment   | :heavy_check_mark: | :heavy_check_mark: | :heavy_multiplication_x: | :heavy_check_mark: | :heavy_check_mark: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [Trusted Platform Module](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#trusted-platform-module-tpm).  Please review the [samples](./provisioning/device/samples/) folder on how to create a device client.    Websocket connection over AMQP is in progress.                                                                                                                                                                                                        |
+| X.509 Individual Enrollment | :heavy_check_mark: | :heavy_check_mark:* | :heavy_check_mark: | :heavy_check_mark:* | :heavy_check_mark: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [X.509 root certificate](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#root-certificate).  Please review the [samples](./provisioning/device/samples/) folder on how to create a device client.    Websocket connection with X.509 on .NETCore is available on Linux but not on Windows.  For Windows, please use .Net instead.  |
```

Comment:
>instead [](start = 655, length = 7)

Also, Chris will provide you a link to the blocking issue in .NETCore.

Reply:
Please reference our dependency here: https://github.com/dotnet/corefx/wiki/ApiCompat#clientwebsocketoptions
The bug is within the wiki.